Glyssen/Dialogs/SplitBlockDlg.cs

```diff
@@ -24,7 +24,7 @@ public partial class SplitBlockDlg : FormWithPersistedSettings
 		private int m_blockSplitIdCounter = 1;  // zero is reserved for assigning a character id to the first segment
 		private readonly IEnumerable<AssignCharacterViewModel.Character> m_characters;
 		private readonly string m_css = Resources.BlockSplitCss;
-		private readonly Regex m_verseNumberRegex = new Regex(@"<sup>.*</sup>");
+		private readonly Regex m_verseNumberRegex = new Regex(@"(<span.*?>.+?<\/span>)+<sup>.*?<\/sup>");
```

Comment:
I'm not clear as to what motivated this change. It looks like you're saying that any (potentially nested) spans immediately preceding the super-scripted text should also be considered part of the verse number. But why?